frontend/src/views/SupplierDashboard.vue

```diff
@@ -129,122 +157,235 @@
 </template>
 
 <script>
+import supplierDashboardMockData from "@/assets/supplierDashboardMockData.json";
+import UnitOfMeasureUtils from "@/services/UnitOfMeasureUtils";
 
 export default{
   name: "SupplierDashboard",
 
   data() {
     return{
+        // get environmental endpoints
+        backendURL: import.meta.env.VITE_BACKEND_BASE_URL,
+        backendApiKey: import.meta.env.VITE_BACKEND_API_KEY,
+        endpointProducts: import.meta.env.VITE_ENDPOINT_PRODUCTS,
+        endpointCustomers: import.meta.env.VITE_ENDPOINT_CUSTOMER,
+        endpointAllProductStocks: import.meta.env.VITE_ENDPOINT_PRODUCT_STOCKS,
+        endpointReportedProductStocks: import.meta.env.VITE_ENDPOINT_REPORTED_PRODUCT_STOCKS,
         // Customer dropdown
         dropdownCustomer: "",
         // Material dropdown
-        dropdownMaterial: "",
+        dropdownProducts: "",
+        // BPNS dropdown
+        dropdownBPNS: "",
```

Comment:
Please bind to customer's sites and addresses.

Reply:
The sites are already bound as an object into the dropdownBPNS variable. Same applies to the addresses and the dropdownBPNA variable.